custom_model_runner/datarobot_drum/resource/components/Python/prediction_server/prediction_server.py

```diff
@@ -115,27 +115,8 @@ def stats():
             self._stats_collector.stats_reset()
             return ret_dict, HTTP_200_OK
 
-        def _shutdown_server():
-            func = request.environ.get("werkzeug.server.shutdown")
-            if func is None:
-                raise RuntimeError("Not running with the Werkzeug Server")
-            func()
-
-        @app.route("{}/".format(url_prefix))
-        def ping():
-            """This route is used to ensure that server has started"""
-            return "Server is up!\n", HTTP_200_OK
-
-        def _shutdown_server():
-            func = request.environ.get("werkzeug.server.shutdown")
-            if func is None:
-                raise RuntimeError("Not running with the Werkzeug Server")
-            func()
-
-        @app.route("{}/shutdown/".format(url_prefix), methods=["POST"])
-        def shutdown():
-            _shutdown_server()
-            return "Server shutting down...", HTTP_200_OK
+        app = get_flask_app(model_api)
+        logging.getLogger("werkzeug").setLevel(logger.getEffectiveLevel())
```

Comment:
Eventually, looks like, this affects only on printing log messages in terminal.
Like: > received, response 200
you know what i mean...